src/mame/drivers/midqslvr.cpp

```diff
@@ -2,25 +2,254 @@
 // copyright-holders:Angelo Salese
 /***************************************************************************
 
-    Midway Quicksilver skeleton driver
-
-    TODO:
-    - offrthnd: illegal opcode tripped just after that PIIX4 is recognized
-
-    Main CPU : Intel Celeron 333/366MHz
-    Motherboard : Intel SE440BX-2
-    RAM : 64MB PC100-222-620 non-ecc
-    Sound: Integrated YMF740G
-    Networking: SMC EZ Card 10 / SMC1208T (probably 10ec:8029 1113:1208)
-    Graphics Chips : Quantum Obsidian 3DFX
-    Storage : Hard Drive
-
-    Chipsets (440BX AGPset):
-    - 82371EB PCI-ISA bridge
-    - 82371EB Power Management Controller
-    - 82371AB/EB Universal Host Controller (USB UHCI)
-    - 82371AB/EB PCI Bus Master IDE Controller
-
+Midway Quicksilver II/Graphite skeleton driver
+	
+Hardware configurations:
+
+Hydro Thunder:   Quicksilver II system and Diego I/O board
+Offroad Thunder: Quicksilver II system and Magicbus I/O board	
+Arctic Thunder:  Graphite system and Substitute I/O board
+	
+All of the games communicate with their I/O boards serially.
+
+Quicksilver II hardware:
+- Main CPU: Intel Celeron 333/366MHz
+- Motherboard: Intel SE440BX-2
+- RAM: 64MB PC100-222-620 non-ecc
+- Sound: Integrated YMF740G
+- Networking: SMC EZ Card 10 / SMC1208T (probably 10ec:8029 1113:1208)
+- Video Card: Quantum Obsidian 3DFX Voodoo 2 (CPLD protected)
+- Storage: Hard Drive
+- OS: TNT Embedded Kernel
+	
+Note: This was once claimed to run on Windows 95 or 98 but has been proven (mostly) false. The TNT Kernel was a "DOS Extender"
+that allows core Windows NT functions to work on MS DOS. It's also possible it runs on a custom made OS as both games do not display
+anything DOS related. Both games also have their video signals from their respective I/O boards, EGA res, as opposed to the video card itself.
```

Comment:
What do you mean by "EGA res"?  EGA was a standard for digital input monitors.  These boards output analog video.  Is it 15kHz standard resolution, 31kHz medium resolution, or something else?

Reply:
15khz is standard res, . 24khz is medium res. 31khz is VGA res. A medium resolution cannot display an image from a standard resolution source and vice versa.